cmake/modules/OCBundleResources.cmake

```diff
@@ -0,0 +1,166 @@
+function(__add_file_to_qrc_file)
+    set(options "")
+    set(oneValueArgs QRC_PATH FILE_PATH ALIAS)
+    set(multiValueArgs)
+    cmake_parse_arguments(__ADD_FILE_TO_QRC_FILE "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
+
+    foreach(param QRC_PATH FILE_PATH)
+        if(NOT __ADD_FILE_TO_QRC_FILE_${param})
+            message(FATAL_ERROR "${CMAKE_CURRENT_FUNCTION}: Argument missing: ${param}")
+        endif()
+    endforeach()
+
+    set(line "<file ")
+    if(__ADD_FILE_TO_QRC_FILE_ALIAS)
+        set(line "${line} alias=\"${__ADD_FILE_TO_QRC_FILE_ALIAS}\"")
+    endif()
+    set(line "${line}>${__ADD_FILE_TO_QRC_FILE_FILE_PATH}</file>")
+
+    file(APPEND ${__ADD_FILE_TO_QRC_FILE_QRC_PATH} "        ${line}\n")
+endfunction()
+
+
+function(__addIcon QRC_PATH THEME ICON_NAME)
+    set(options)
+    set(oneValueArgs SRC_PATH)
+    set(multiValueArgs)
+    cmake_parse_arguments(_ICON "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
+
+    if(NOT _ICON_SRC_PATH)
+        set(_ICON_SRC_PATH ${THEME})
+    endif()
+
+    set(icon "theme/${_ICON_SRC_PATH}/${ICON_NAME}.svg")
+    set(iconAlias "theme/${THEME}/${ICON_NAME}.svg")
+    if (EXISTS ${OEM_THEME_DIR}/${icon})
+        file(APPEND "${QRC_PATH}" "<file alias=\"${iconAlias}\">${OEM_THEME_DIR}/${icon}</file>\n")
+    else()
+        set(icon "theme/${_ICON_SRC_PATH}/${ICON_NAME}.png")
+        set(iconAlias "theme/${THEME}/${ICON_NAME}.png")
+        if (EXISTS ${OEM_THEME_DIR}/${icon})
+            __add_file_to_qrc_file(
+                QRC_PATH ${QRC_PATH}
+                FILE_PATH ${OEM_THEME_DIR}/${icon}
+                FILE_ALIAS ${iconAlias}
+            )
+        else()
+            set(SIZES "16;22;32;48;64;128;256;512;1024")
+            foreach(size ${SIZES})
+                set(icon "theme/${_ICON_SRC_PATH}/${ICON_NAME}-${size}.png")
+                set(iconAlias "theme/${THEME}/${ICON_NAME}-${size}.png")
+                if (EXISTS ${OEM_THEME_DIR}/${icon})
+                    __add_file_to_qrc_file(
+                        QRC_PATH ${QRC_PATH}
+                        FILE_PATH ${OEM_THEME_DIR}/${icon}
+                        FILE_ALIAS ${iconAlias}
+                    )
+                endif()
+            endforeach()
+        endif()
+    endif()
+endfunction()
+
+function(__write_qrc_file_header QRC_PATH FILES_PREFIX)
+    file(WRITE ${QRC_PATH} "<RCC>\n")
+    file(APPEND ${QRC_PATH} "    <qresource prefix=\"/client/${APPLICATION_SHORTNAME}\">\n")
```

Comment:
Hmm the files are bot available through the alias and the actual path.
Maybe we can make the actual path make look less official?
![image](https://user-images.githubusercontent.com/200626/127108916-bd144af9-ca34-4b27-9822-b587d5f7e1fb.png)

Reply:
Looks like Qt creator's rendering is suboptimal there. Gammaray shows the correct paths, as expected.

I removed the shortname because translations are not brand-specific.